package.json

```diff
@@ -24,18 +24,17 @@
   ],
   "scripts": {
     "build": "yarn build:cjs && yarn build:esm",
-    "build:cjs": "yarn workspaces run babel src -d dist --ignore '**/*.test.js,**/*.test.jsx,**/*.stories.js,**/*.stories.jsx' --root-mode upward",
-    "build:esm": "BABEL_TYPE=esm yarn workspaces run babel src -d esm --ignore '**/*.test.js,**/*.test.jsx,**/*.stories.js,**/*.stories.jsx' --root-mode upward",
+    "build:cjs": "yarn workspaces foreach --no-private exec babel src -d dist --ignore '**/*.test.js,**/*.test.jsx,**/*.stories.js,**/*.stories.jsx' --root-mode upward",
```

Comment:
Am I right in saying that `--no-private` here is exclusively to prevent these scripts running for the root package.json?

Reply:
Yep that's right